orchagent/muxorch.cpp

```diff
@@ -1099,9 +1099,6 @@ void MuxOrch::updateRoute(const IpPrefix &pfx, bool add)
 
     if (!add)
     {
-        SWSS_LOG_INFO("Removing route %s from mux_multi_active_nh_table",
```

Comment:
Can you please keep the log to NOTICE and update log message with pfx details since its returning from here?